core/client/src/client.rs

```diff
@@ -1715,8 +1774,14 @@ pub(crate) mod tests {
 		let client = test_client::new();
 
 		let genesis_hash = client.info().unwrap().chain.genesis_hash;
+		let longest_chain_select = test_client::client::LongestChain::new(
+			client.backend().clone(),
+			client.import_lock()
+		);
+
 
-		assert_eq!(genesis_hash.clone(), client.best_containing(genesis_hash.clone(), None).unwrap().unwrap());
+		assert_eq!(genesis_hash.clone(), longest_chain_select.best_containing_for_authoring(
```

Comment:
@DemiMarie-parity is this not the test that you were asking for?

Reply:
It is, I just didn’t see it.